### PR TITLE
add vc example + minor fixes

### LINKED ISSUE #9: Example verifiable credential
It would be helpful to include an example of a full verifiable credential that uses the example schema.

### PR DESCRIPTION
**Pull request by glcohen**
#9

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -537,13 +540,44 @@ By introducing a [=modelVersion=] field we allow the [=credential schema=] to be
 
 These are just a few examples that illustrate the flexibility of the proposed model. It can be extended to support a wide variety of use-cases and make the burden on issuance and verification simpler by facilitating the development of higher-level tooling.
 
+# Examples # {#examples}
+
+## Verifiable Credentials ## {#vc_example}
+
+The following example shows a reference to a [=Credential Schema=] with the identifier <b></b> instide of a Verifiable Credential following the [[VC_DATA_MODEL]]. The example is adopted from <a href="https://w3c.github.io/vc-data-model/#example-18-usage-of-the-credentialschema-property-to-perform-json-schema-validation">Example 18</a> in the specification.
```
Comment (OR13) [inline spec/index.bs]: "instide" -> "inside"

File: spec/index.bs [hunk 2]
```diff
@@ -537,13 +540,44 @@ By introducing a [=modelVersion=] field we allow the [=credential schema=] to be
...
+  "type": ["VerifiableCredential", "ExampleCredential"],
+  "issuer": "https://example.edu/issuers/565049",
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "credentialSchema": {
+    "id": "did:example:c276e12ec21ebfeb1f712ebc6f1;id=83644e73-f540-443c-a440-4d0252dc4f1c;version=1.0",
+    "type": "ExampleSchemaValidator"
+  },
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "exampleKey": "exampleValue"
```
Comment (OR13) [inline spec/index.bs]: I think we need to include the schema and credential in this example, and try to provide a more specific example like emailAddress or ideally something more reflective of the workday use case.

Comment (glcohen) [reply spec/index.bs]: updated - is that what you intended?

Comment (OR13) [reply spec/index.bs]: yep, this is better.

**Pull request merged**